### PR TITLE
#347 create attachments card

### PR DESCRIPTION
**Pull request by fabienmoyon**
Capture d’écran de 2021-04-15 19-48-27

Capture d’écran de 2021-04-19 15-32-12
Capture d’écran de 2021-04-19 15-31-35

### CODE REVIEW

**Comment by chibenwa** [comment]
Ho I see a download all button ;-)

Is it m surprise?

Canyou post a screenshot with one attahment only? With 2 attachments?

**Comment by fabienmoyon** [comment]
> Ho I see a download all button ;-)
> Is it m surprise?

Download buttons are not in this PR, it was just to see the final result when I added the feature. 

> Can you post a screenshot with one attahment only? With 2 attachments?

yes, done

File: src/linagora.esn.unifiedinbox/views/partials/email.pug [hunk 1]
```diff
@@ -36,8 +36,20 @@
         message="email"
       )
 
-    .lv-footer.attachments(ng-if="email.attachments.length > 0")
-      inbox-attachment(ng-repeat="attachment in email.attachments", ng-if="::!attachment.isInline", attachment="::attachment")
+    div(ng-if="email.attachments.length > 0")
+      md-card
+        md-card-header(ng-click="isExpand = isExpand ? false : true")
+          md-card-header-text.flex.flex-start
+            i.mdi.mdi-paperclip(title="{{ attachmentsNumber > 1 ? 'Attachments' : 'Attachment' | translate }}")
```
Comment (oubchid) [inline src/linagora.esn.unifiedinbox/views/partials/email.pug:43]: Maybe you can use `ngPluralize` to pluralize and format things

File: src/linagora.esn.unifiedinbox/views/partials/email.pug [hunk 2]
```diff
@@ -36,8 +36,20 @@
         message="email"
       )
 
-    .lv-footer.attachments(ng-if="email.attachments.length > 0")
-      inbox-attachment(ng-repeat="attachment in email.attachments", ng-if="::!attachment.isInline", attachment="::attachment")
+    div(ng-if="email.attachments.length > 0")
+      md-card
+        md-card-header(ng-click="isExpand = isExpand ? false : true")
```
Comment (alagane) [inline src/linagora.esn.unifiedinbox/views/partials/email.pug:41]: Can you use `!` instead? I think it is easier to understand.
```suggestion
        md-card-header(ng-click="isExpand = !isExpand")
```

File: src/linagora.esn.unifiedinbox/views/partials/email.pug (hunk 1 above)
Comment (alagane) [reply src/linagora.esn.unifiedinbox/views/partials/email.pug:43]: Seems to be a good idea, though it seems ngPluralize is not used anywhere in the app.

Comment (fabienmoyon) [reply src/linagora.esn.unifiedinbox/views/partials/email.pug:43]: Yes, exactly @alagane and it works with a ternary so let keep it like this but interesting I was not aware of this directive!

**Review by alagane — APPROVED** [review]

**Review by oubchid — APPROVED** [review]

**Comment by chibenwa** [comment]
Looks good to me

**Pull request merged**